src/main/java/org/jabref/gui/SidePaneComponent.java

```diff
@@ -88,7 +90,7 @@ public final Node getContentPane() {
     /**
      * @return the header pane for this component
      */
-    public final Node getHeader() {
+    public Node getHeader() {
```

Comment:
Instead of overriding this method, I would propose to create a new (overridable) method `getHeaderButtons` that results additional header button to be displayed. In this way you ensure consistency as well as reduce code duplication.

Reply:
Added a protected method for gettingAddtionalHeader buttons